### PR TITLE
configured CI/CD pipeline

### PR DESCRIPTION
**Pull request by vickyteng**
will add documentation soon

### CODE REVIEW

File: client/.env.sample [hunk 1]
```diff
@@ -0,0 +1 @@
+NODE_PATH=src/
```
Comment (juanflo) [inline client/.env.sample]: Should add port here too

File: client/src/socket/index.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+import io from 'socket.io-client'
+
+const socket = io.connect('https://texasplan-em-server.herokuapp.com/');
```
Comment (juanflo) [inline client/src/socket/index.js]: The server should be in a environment variable.

File: CICD.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+### Deploying to Heroku
+1. Create a new Heroku application, and push your application to a Git remote repository
+
+  ```
+  $ heroku create
```
Comment (wongm3) [inline CICD.md]: probably want to add that you should specify a name or heroku will generate a random one for you

**Comment by vickyteng** [comment]
Closing this PR as it is my last day. Ani will make a new pull request.

**Pull request closed without merging**